tokenizers/src/utils/trie.rs

```diff
@@ -0,0 +1,301 @@
+use std::collections::HashMap;
+use std::hash::Hash;
+
+#[derive(Default)]
+pub struct TrieBuilder<Label> {
+    trie: Trie<Label>,
+}
+
+impl<Label: Eq + Hash + Copy> TrieBuilder<Label> {
+    pub fn push(&mut self, element: &[Label]) {
+        self.trie.push(element);
+    }
+
+    pub fn build(self) -> Trie<Label> {
+        self.trie
+    }
+}
```

Comment:
This feels redundant, wouldn't a `Trie::new(elmts: &[Label])` be cleaner?

Reply:
Hmm yes, thought the same, it's old code, but can probably remove.